.github/workflows/build.yml

```diff
@@ -0,0 +1,49 @@
+name: MicroProfile Config Build
```

Comment:
If we add this, would we want to remove the hook that automatically requests a Jenkins build?

I assume we'd still need to use the MP Jenkins to build an actual release?

Reply:
We still request a Jenkins build. With this, we can get our PRs to build against different jvm and get good logs.

Reply:
Update the typo from Config to Fault Tolerance.